Source/WebCore/css/CSSValuePool.cpp

```diff
@@ -149,12 +115,9 @@ RefPtr<CSSValueList> CSSValuePool::createFontFaceValue(const AtomString& string)
     if (m_fontFaceValueCache.size() >= maximumFontFaceCacheSize)
         m_fontFaceValueCache.remove(m_fontFaceValueCache.random());
 
-    return m_fontFaceValueCache.ensure(string, [&string] () -> RefPtr<CSSValueList> {
-        auto result = CSSParser::parseSingleValue(CSSPropertyFontFamily, string);
-        if (!is<CSSValueList>(result))
-            return nullptr;
-        // FIXME: Make downcast work on RefPtr, remove the get() below, and save one reference count churn.
```

Comment:
Why remove the FIXME?

Reply:
It’s not particularly valuable. I wrote that FIXME, and we really don’t need a comment each place we use downcast that mentions we should make a version that doesn't churn reference counts. It’s still an OK idea, but not super important.